src/scss/components/_listings.scss

```diff
@@ -50,21 +80,149 @@ $listing-borderColor: $color-border;
             }
         }
 
+        .c-listing-item-img {
+            float: left;
+            width: 50px;
+
+            @include media('>wide') {
+                width: 65px;
+            }
+        }
+
+        .c-listing-item-imgItem {
+            width: 100%;
+            height: auto;
+        }
+
         .c-listing-item-info {
+            top: -4px;
+            position: relative;
             color: $color-text;
-            flex-wrap: nowrap;
-            margin-left: spacing(base);
+            padding-left: 66px;
+
+            @include media('>wide') {
+                position: relative;
+                padding-left: 80px;
+                padding-right: 260px;
+
+                &:after {
+                    content: '';
+                    top: 4px;
+                    width: 1px;
+                    right: 259px;
+                    height: 100%;
+                    display: block;
+                    position: absolute;
+                    background: $listing-borderColor;
+
+                    .c-listing--offline & {
+                        background: $listing-offlineBorderColor;
+                    }
+                }
+            }
+        }
+
+        .c-listing-item-title {
+            word-break: break-word;
         }
+
+        .c-listing-item-meta {
```

Comment:
Would be good to comment this, as to what it refers to.  Not obvious from `meta` – guessing it's some sort of information in the listing item?